src/mongo/db/auth/security_file.cpp

```diff
@@ -53,11 +53,35 @@ StatusWith<std::string> readSecurityFile(const std::string& filename) {
     }
 
 #if !defined(_WIN32)
-    // check permissions: must be X00, where X is >= 4
-    if ((stats.st_mode & (S_IRWXG | S_IRWXO)) != 0) {
+    if (stats.st_uid == 0)
+    {
```

Comment:
MongoDB code base always places opening curly braces on the same line with statement. Please follow this rule.